bundler/lib/bundler/cli/gem.rb

```diff
@@ -183,6 +193,12 @@ def run
                         "so -t is not needed if you want to continue using it. " \
                         "This setting can be changed anytime with `bundle config gem.test`."
       end
+
+      if options[:ci] == Bundler.settings["gem.ci"]
```

Comment:
What about:
```ruby
"#{options[:ci]} is already configured, ignoring --ci flag"
```

Reply:
The current reminder text appears after the gem has been generated. I put it towards the end so that it would be easier to notice once all the generator output is shown.

If we moved this hint to before any templates are generated, I think your suggestion would work better. What do you think?

Reply:
Sounds good 👍